proto/nodeinstance/info/v1/info.proto

```diff
@@ -33,8 +37,29 @@ message UpdateNodeInfo {
   bool child = 6;
 
   MachineLearningInfo ml_info = 8;
+  // to be obsoleted in far future
+
+  NodeInfo2 node_info = 9;
+  // node_info shows data about actual node
+  // for example feature (ml) for this
+  // node (child) might be available/enabled on the node (child) directly
+  // but not available trough the parent (node_instance)
+
+  NodeInstanceInfo node_instance_info = 10;
+  // info specific to the node_instance for this node available trough agent
+  // who sends this message.
+  // e.g. machine learning is enabled for this node and processing is done
+  // by the actual agent (parent). (child itself might or might not be
+  // ml ml_capable by itself (see node_info))
 }
 
+message NodeInfo2 {
+  repeated  aclk_lib.v1.Capability capabilities = 1;
+}
```

Comment:
Small question, so when `NodeInfo data = 2;` is obsoleted, then these (both NodeInfo2 and NodeInstanceInfo) should contain also a message of `NodeInfo`, right?

Reply:
not sure I get the question exactly. But NodeInfo2 and NodeInstanceInfo will not contain 'NodeInfo'. 'NodeInfo' fields should be migrated step by step to relevant new message.